packages/0xcert-scaffold/src/core/asset-ledger.ts

```diff
@@ -343,9 +357,16 @@ export interface AssetLedgerObjectUpdateRecipe {
 export interface AssetLedgerUpdateRecipe {
 
   /**
-   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended foo each asset.
+   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended for each asset.
```

Comment:
* URI base for metadata URIs. At the end of the base, the assetId is automatically appended for each asset.